src/app/controllers/MachineController.ts

```diff
@@ -568,6 +568,12 @@ export default class MachineController {
 			const unixTime = this.findAnyData(sortedVal, sortedNode, sortedSystem, (value) => {
 				return value.timestamp
 			})
+
+			// do not display machines that have not been online for more than 4 days
+			if (unixTime && Date.now() - unixTime > 96 * 60 * 60 * 1000) {
```

Comment:
You could save the return value of `Date.now()` once above the for loop. Feel free to ignore.